reflex/components/plotly/plotly.py

```diff
@@ -33,10 +33,10 @@ class Plotly(PlotlyLib):
     layout: Var[Dict]
 
     # The config of the graph.
-    config: Var[Dict]
+    # config: Var[Dict]
```

Comment:
does this really not work? these are the parameters that `config` is expected to take https://github.com/plotly/plotly.js/blob/7ffcdd7d602d52e8a0e75d766ae18fe8e9eb54fb/src/plot_api/plot_config.js#L15